Added 'Message' parameter to DBotScore (#1028)
demisto/content-docs: docs/integrations/dbot.md

} 
```

The DBot score must be at the root level of the context and contain **all** the required keys, as listed below.

| Key | Meaning | Required
| --- | --- | --- |
| Indicator | The indicator value. | Required |
| Type | The indicator type. Can be: ip, file, email, url, cve, account, cider, domainglob, certificate, or cryptocurrency. | Required |
| Vendor | The vendor reporting the score of the indicator.| Required |
| Score | An integer regarding the status of the indicator. See [Score Types](#score-types) below.| Required |
| Reliability | The reliability of the source providing the intelligence data. See [Reliability Level](#reliability-level) below.| Optional |

> | Message | Used to message on the API response. For example, when the return API response is `"Not found"`. | Optional |

Reviewer: ```suggestion
| Message | Optional message to show an API response. For example, `"Not found"`. | Optional |
```
